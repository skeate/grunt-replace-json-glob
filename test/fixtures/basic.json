{
    "files": ["test/fixtures/*.json"],
    "other": "string",
    "anotherProperty": false
}
