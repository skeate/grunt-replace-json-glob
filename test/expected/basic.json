{
  "files": [
    "test/fixtures/basic.json",
    "test/fixtures/subdir.json"
  ],
  "other": "string",
  "anotherProperty": false
}
